### ISSUE TITLE
create_index changes its argument

### ISSUE THREAD

**Issue by larets** [tarantool/tarantool#2480]
```
tarantool> xuu
—-
- - 1
  - num
  - 2
  - num
...

tarantool> box.space.pzds:create_index('fifs', {parts=xuu})
—-
- unique: true
  parts:
  - type: NUM
    fieldno: 1
  - type: NUM
    fieldno: 2
  id: 5
  space_id: 555
  name: fifs
  type: TREE
...

tarantool> xuu
—-
- - 0
  - num
  - 1
  - num
```

kinda annoying pls fix

**Comment by larets** [comment]
a workaround is {parts={unpack(xuu)}} but hey

**Comment by kostja** [comment]
* larets <notifications@github.com> [17/05/26 16:35]:


Please specify your Tarantool version. I believe this is fixed in
1.7.4

-- 
Konstantin Osipov, Moscow, Russia, +7 903 626 22 32
http://tarantool.org - www.twitter.com/kostja_osipov

**Comment by larets** [comment]
Yep my tarantool installed via pip was 1.6.7, checked that it's fixed in 1.7.4
